### DISCUSSION TITLE
How to verify my password?

### DISCUSSION THREAD

**Discussion by quazgar** [borgbase/vorta#1920] [FAQ]
Is there a way to check if I still know the repo's password?  As far as I can tell, once a repo has been set up, all the connections use the stored password.  For normal usage, that's 100% what I want.

But in order to verify an existing setup, I would like to be able to

1. enter the repository's passwort manually
2. maybe get a copy&pasteable command line, with all paths and settings, for something like `borg list --rhs "ssh -i ..." ssh://...@.../.../.../`.  Then I could save this line to some secure location, to have a starting point if I ever need to restore the backup without all the fancy Vorta GUI.

Is there a way to do these? So far the only way I see is to open the SQL file with sqlitebrowser and extract the settings from there.  The closest I could find was the "copy to clipboard" button in the "repository" tab which at least gives the remote URL.

https://github.com/borgbase/vorta/discussions/821#discussioncomment-369306 goes in a similar direction, but is not quite the same.

**Comment by goebbe** [comment]
@quazgar  My understanding is that there are **different types of password/ passphrases** involved: 

a) the username, url, **password of the remote backup server**  - this is used for the setup, e.g. to copy the .pub ssh key to the remote server.
b) **ssh keys** (private/ public keys) - this is used by Vorta to securely connect to the remote backup server. 
c) the **passphrase of the repository** - this is used in order to encrypt/ decrypt your repositories.

Where can we find the passwords: 
a) the **password for the server** is provided by the company / administrator of the remote backup server
b) **ssh keys** are saved in a hidden directory in your home directory:  ~/.ssh/
c) the **passphrase of the repo** can be exported to a .json file in Vorta on a per-profile base. Choose the profile, click on export, select that the passphrase should be included to the .json file. You can open the .json file, using a text editor and search for the passphrase.
   On **Linux** (Gnome/ Cinnamon) the passphrase of the repositories are stored in the key-ring (libsecrets) of the OS. Install/ open "seahorse" or "Passwords and Keys" in order to check the saved passphrases. 

Hope this helps.
